# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/vamps/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur vamps-0.95/Makefile vamps-0.95-patched/Makefile
--- vamps-0.95/Makefile	2003-12-15 07:55:45.000000000 +0100
+++ vamps-0.95-patched/Makefile	2004-05-18 22:35:37.000000000 +0200
@@ -14,7 +14,7 @@
 
 CC     = gcc
 CFLAGS = -Wall -O3 -fomit-frame-pointer -fexpensive-optimizations \
-         -funroll-loops -march=i686 -mcpu=i686 -malign-functions=4
+        -funroll-loops -march=i686 -mcpu=i686 -falign-functions=4
 STATIC =
 #STATIC = -static
 STRIP  = strip
@@ -23,7 +23,7 @@
 all:			vamps vamps-play_title lsdvd
 
 vamps:			vamps.o requant.o
-			$(CC) -o $@ $(STATIC) vamps.o requant.o -lpthread
+			$(CC) -o $@ $(STATIC) vamps.o requant.o -lpthread -lm
 			$(STRIP) $@
 
 requant.o:		requant.h putvlc.h getvlc.h
Nur in vamps-0.95-patched: Makefile~.
